### DISCUSSION TITLE
中國研發 AI 檢查官

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#278] [General]
中國科學家 2015~2020 年使用約 1.7 萬件真實案例訓練這套工具，現稱 System 206，中國最大區檢察院上海浦東人民檢察院測試，號稱可辨識起訴上海最常見的八種犯罪行為，如:信用卡詐騙、經營賭博、危險駕駛、故意傷害、妨礙公務、盜竊、欺詐和尋釁滋事。
然而，長期以來 AI 執法一直受政治活動家和工程師等批評，認為現有技術的細微差別並不足以伸張正義，直言這是連人類都尚未掌握的領域，看來中國的System 206仍需經過種種考驗。

[中國AI檢察官，號稱可以口述定罪](https://technews.tw/2021/12/29/ai-judgment-in-china/)

**Comment by wax75397** [comment]
雖然中國對於人民的管控手段十分不可取，但也挺佩服它們對於科技的應用與想法。

**Reply by HSU-YU-JEN** [reply]
沒錯~但感覺法律判決這種事情還是要經過人類處理會比較恰當

**Comment by MAKERRRofKEVIN** [comment]
之前申請法律系的時候看過一本明日世界的律師，作者有預測過未來這類科技輔助判決會越來越常見，如果不完全移掉現有的司法人員讓機器過度介入的話，兩者交互合作確實能夠一方面維持判決的公信力一方面又能大大提高效率，這確實是個值得期待的好應用

**Comment by yt7l0** [comment]
但是法官的判決通常會有「人情」的考量，僅靠AI判斷似乎有點太過決斷
